src/shared/parameter_v2/config_definitions/corner_kick_play_config.yaml

```diff
@@ -0,0 +1,6 @@
+- double:
```

Comment:
What if we put all of these into a playconfig?

Reply:
yeah that would be a good idea, in this PR im not really reorganizing the params

will do this in the next pass